stats/integration_test.go

```diff
@@ -20,9 +23,132 @@ import (
 
 var privateKey interface{}
 
+func TestContainerStats(t *testing.T) {
+	dialer := &websocket.Dialer{}
+	headers := http.Header{}
+	c, err := client.NewClient("unix:///var/run/docker.sock")
+	if err != nil {
+		t.Fatalf("Could not connect to docker, err: [%v]", err)
+	}
+
+	ctrs, err := c.ListContainers(client.ListContainersOptions{
+		Filters: map[string][]string{
+			"image": []string{"google/cadvisor"},
+		},
+	})
+	if err != nil || len(ctrs) == 0 {
+		t.Fatalf("Error listing all images, err : [%v]", err)
+	}
+
+	cIds := map[string]string{}
+	payload := map[string]interface{}{
+		"hostUuid":     "1",
+		"containerIds": cIds,
+	}
+
+	for i, ctr := range ctrs {
+		cIds[ctr.ID] = "1i" + strconv.Itoa(i+1)
+	}
+
+	token := wsp_utils.CreateTokenWithPayload(payload, privateKey)
+	url := "ws://localhost:1111/v1/containerstats?token=" + token
+	ws, _, err := dialer.Dial(url, headers)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer ws.Close()
+
+	for count := 0; count < 4; count++ {
+		_, msg, err := ws.ReadMessage()
+		if err != nil {
+			t.Fatal(err)
+		}
+		stats := string(msg)
+		if !strings.Contains(stats, "1i1") || !strings.Contains(stats, "1i2") {
```

Comment:
The way this is written, it doesn't actually test the bug we're fixing, right? To fully assert the fix, we'd have to ensure that we see both 1i1 and 1i2, not either.